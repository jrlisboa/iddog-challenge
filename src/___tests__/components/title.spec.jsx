import React from 'react';
import { shallow } from 'enzyme';
import Title from '../../components/title';

import {
    TitleStyle,
} from '../../components/title/index.style';

describe('Title component', () => {
    const component = shallow(<Title />);
    it('Should render Title component without crash', () => {
        expect(component).toBeTruthy();
    });

    it('Should match snapshot', () => {
        expect(component).toMatchSnapshot();
    });

    it('shold render all visual components', () => {
        expect(component.find(TitleStyle)).toHaveLength(1);
    });
});
