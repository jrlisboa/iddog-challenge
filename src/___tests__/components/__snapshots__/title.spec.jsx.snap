// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component Should match snapshot 1`] = `ShallowWrapper {}`;
